### DISCUSSION TITLE
Point blank + Rigtools ??

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1224] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Rigtools can run on chrome webstore or extensions pages  can I combine it with point blank to be able to run on these pages

**Comment by schoolexploitkid** [comment]
it can, but rigtools already turns extensions off, so what's the purpose?

**Comment by akabutnicer** [comment, marked as answer]
The Chrome webstore has been gaslighted so much. You can't install any arbitrary extensions with the Chrome Webstore. What you should be looking at is `chrome://extensions`. Find point blank on there and then we have something.
